src/test/java/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,54 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class StringAddCalculatorTest {
+
+    @DisplayName("빈 문자열 또는 null 값을 입력할 경우 0을 반환")
+    @Test
+    public void calculate_null_또는_빈문자() {
+        int result = StringAddCalculator.calculate(null);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAddCalculator.calculate("");
+        assertThat(result).isEqualTo(0);
```

Comment:
```suggestion
    assertThat(StringAddCalculator.splitAndSum(null)).isEqualTo(0);
    assertThat(StringAddCalculator.splitAndSum("")).isEqualTo(0);
```

- 변수명을 재활용하면 추후 유지보수할때 혼란을 야기시킬수 있는 포인트가 될수도 있다고 생각합니다~
- 좀 더 명확한 테스트 코드를 고민해보아요~
- assertAll을 사용해도 좋겠네요~
- @NullAndEmptySource 활용해볼수도 있겠네요!